drizzlepac/mapreg.py

```diff
@@ -1063,9 +1061,6 @@ def _check_FITS_extensions(img, extensions):
     return all_present
 
 
-#--------------------------
-# TEAL Interface functions
```

Comment:
Again, I do not think it hurts to keep the comments for context though you might modify the comment to indicate the TEAL interactive GUI is "disabled"or hidden, but some underlying functionality will remain (or something like this).